{"uuid": "20fffc51-c518-4479-881f-1ba82fd2de9a", "children": ["8ca28667-bb9d-4f05-afaf-82216875635c"], "befores": [{"name": "_Class__pytest_setup_method", "status": "broken", "statusDetails": {"message": "AttributeError: 'TestCustomerManageEdit' object has no attribute 'go_to_customer_manage_page'\n", "trace": "  File \"D:\\softwore\\python372\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"D:\\softwore\\python372\\lib\\site-packages\\_pytest\\fixtures.py\", line 923, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"D:\\softwore\\python372\\lib\\site-packages\\_pytest\\fixtures.py\", line 778, in call_fixture_func\n    res = next(it)\n  File \"D:\\softwore\\python372\\lib\\site-packages\\_pytest\\python.py\", line 773, in xunit_setup_method_fixture\n    _call_with_optional_argument(func, method)\n  File \"D:\\softwore\\python372\\lib\\site-packages\\_pytest\\python.py\", line 691, in _call_with_optional_argument\n    func()\n  File \"D:\\my-git-project\\91lng\\selenium_91lng_auto_test\\test_case\\test_bpm\\test_customer_manage.py\", line 59, in setup_method\n    self.go_to_customer_manage_page()\n"}, "start": 1553478149275, "stop": 1553478158230}], "start": 1553478149275, "stop": 1553478158238}